packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -70,13 +70,14 @@ class NestedScrollView extends StatefulWidget {
   /// Defaults to false.
   final bool reverse;
 
-  /// How the scroll view should respond to user input.
+  /// How the outer scroll view should respond to user input.
   ///
-  /// For example, determines how the scroll view continues to animate after the
-  /// user stops dragging the scroll view.
```

Comment:
You removed this sentence. Maybe consider bringing it back but having it point explicitly to [ScrollPhysics.createBallisticSimulation], which is the main method that one might want to override.